src/multiple_shooting.jl

```diff
@@ -126,6 +146,118 @@ function multiple_shoot(
     )
 end
 
+function multiple_shoot(
+    p::AbstractArray,
+    ode_data::AbstractArray,
+    tsteps::AbstractArray,
+    ensembleprob::EnsembleProblem,
+    ensemblealg::SciMLBase.BasicEnsembleAlgorithm,
+    loss_function::Function,
+    continuity_loss::Function,
```

Comment:
This stops specialization BTW

Reply:
I have removed  the type specification `::Function`. Is that what you had in mind?